### DISCUSSION TITLE
Lay-Z-Spa Tahiti; BESTWAY 58323; Pump Model #54186; 6-Pin; feedback

### DISCUSSION THREAD

**Discussion by Dellonb** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#111] [Q&A]
While waiting for PCB delivery from JLCPCB.com I have built prototype board in order to check if my pump is compatible with the project. 
ESP8266 NodeMCU 1.0 downloaded circuits connected and finally I can manage my spa locally from the touch board as well as from WEB browser.
I did not configure MQTT yet.
No need any extra resistors between ESP8266 NodeMCU and 8 channel bidirectional converter
TXS0108E

Testing from yesterday so far only one issue – beeps/sound are not working at all.
Not sure if I miss any wire or there is something I need to change in the configuration ?

I can see that last PIN no 6 is not used on controller CIO/JST SM /female / H1 (towards touch board)

No sound/beep at all when touch the display board buttons or Rebooting ESP
“..Custom sound instead of just beeping is possible.. “ any idea how to start troubleshooting ?

![IMG_20210812_210443](https://user-images.githubusercontent.com/75362771/129415030-20b6ce0a-14df-4cf2-88fe-38ff8bc50c5a.jpg)

![IMG_20210812_210439_1](https://user-images.githubusercontent.com/75362771/129415266-9c10cf68-0ee2-43d5-8059-a000d04c5456.jpg)

**Comment by visualapproach** [comment]
Hi, pin 6 from CIO is not used. We create our own audio in the 8266. Check "audio on" in the config.html page and if it does not work there is probably a short or open circuit somewhere.

**Reply by visualapproach** [reply]
Some people have reported the sound on 2021 models to be at a lower volume. I don't know why that is.

**Reply by Dellonb** [reply]
My Spy controller is now fully working!
All works well as expected.

visualapproach - thanks for a great project.



![001](https://user-images.githubusercontent.com/75362771/131387672-6ac697d5-32a9-40dd-bb65-a4a82bc82b6e.jpg)

**Reply by visualapproach** [reply]
Glad you enjoy it:)
